tools/clang/unittests/HLSLExec/DXRUtil.h

```diff
@@ -0,0 +1,226 @@
+//===------------ DXRUtil.h - DXR Utility Functions ------------*- C++ -*-===//
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+// DXRUtil.h                                                                 //
+// Copyright (C) Nvidia Corporation. All rights reserved.                    //
+// This file is distributed under the University of Illinois Open Source     //
+// License. See LICENSE.TXT for details.                                     //
+//                                                                           //
+//  This file contains the utility functions for DXR execution tests.        //
+//                                                                           //
+///////////////////////////////////////////////////////////////////////////////
+
+//= DXR Utility
+//============================================================================
+#define SHADER_ID_SIZE_IN_BYTES 32
+
+#ifndef ROUND_UP
+#define ROUND_UP(v, powerOf2Alignment)                                         \
+  (((v) + (powerOf2Alignment)-1) & ~((powerOf2Alignment)-1))
+#endif
+struct SceneConsts {
+  DirectX::XMFLOAT4 eye;
+  DirectX::XMFLOAT4 U;
+  DirectX::XMFLOAT4 V;
+  DirectX::XMFLOAT4 W;
+  float sceneScale;
+  unsigned windowSize[2];
+  int rayFlags;
+};
```

Comment:
This needs a pass through to update to follow LLVM naming conventions.

```suggestion
struct SceneConsts {
  DirectX::XMFLOAT4 Eye;
  DirectX::XMFLOAT4 U;
  DirectX::XMFLOAT4 V;
  DirectX::XMFLOAT4 W;
  float SceneScale;
  unsigned WindowSize[2];
  int RayFlags;
};
```

Reply:
Updated to follow coding standards